README.md

```diff
@@ -1383,4 +1383,7 @@ here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-p
 
 I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join community! :smiling_face_with_three_hearts:
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
----
\ No newline at end of file
+---
+#Hi, Kottans!
+I'm Kyrylo, I have always dreamed of programming and this is a great opportunity for me :)
+<img src="https://f3.upet.com/o_XSbhGBwd1G_1.jpg" width="75">
```

Comment:
Hi Kyrylo!
Really good job! 👍 
More preferable is using markdown syntax.  Can you please refactor attached image in something like [this](https://www.markdownguide.org/basic-syntax/#images)
Thanks!

Reply:
HI!
Thanks for the help, I'll fix it now

Reply:
👍